### PR TITLE
Add missing targets to README

### PR DESCRIPTION
**Pull request by networkfusion**
## Description
Update readme with missing boards
Update locations of boards.

## Targets affected

- [ ] BrainPad2
- [ ] GHI_FEZ_CERB40_NF
- [ ] I2M_ELECTRON_NF
- [ ] I2M_OXYGEN_NF
- [ ] MBN_QUAIL
- [ ] NETDUINO3_WIFI
- [ ] PybStick2x
- [ ] ST_NUCLEO64_F401RE_NF
- [ ] ST_NUCLEO64_F411RE_NF
- [ ] ST_STM32F411_DISCOVERY
- [ ] ST_NUCLEO144_F412ZG_NF
- [ ] ST_NUCLEO144_F746ZG
- [ ] ST_STM32F4_DISCOVERY
- [ ] ST_NUCLEO144_F439ZI
- [ ] WEACT_F411CE
- [ ] TI_CC1352P1_LAUNCHXL_868
- [ ] TI_CC1352P1_LAUNCHXL_915
- [ ] BUILD ALL

## Types of changes

- [ ] Improvement (non-breaking change that improves a feature, code or algorithm)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Config and build (change in the configuration and build system, has no impact on code or features)
- [ ] Dependencies (update dependencies and changes associated, has no impact on code or features)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [CONTRIBUTING](https://github.com/nanoframework/.github/blob/master/CONTRIBUTING.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,8 +28,9 @@ The provided boards are **NOT** supported by the .NET **nanoFramework** team and
...
+| [PybStick Std 26](ChibiOS/PybStick2x) | [![Latest Version @ Cloudsmith](https://api-prd.cloudsmith.io/v1/badges/version/net-nanoframework/nanoframework-images-community-targets/raw/PybStick2x/latest/x/?render=true)](https://cloudsmith.io/~net-nanoframework/repos/nanoframework-images-community-targets/packages/detail/raw/PybStick2x/latest/) | STM32F411RE   | 100 MHz   | 512 kB  | 128 kB  |
+| [NETDUINO3_WIFI](ChibiOS/NETDUINO3_WIFI) |  [![Latest Version @ Cloudsmith](https://api-prd.cloudsmith.io/v1/badges/version/net-nanoframework/nanoframework-images-community-targets/raw/NETDUINO3_WIFI/latest/x/?render=true)](https://cloudsmith.io/~net-nanoframework/repos/nanoframework-images-community-targets/packages/detail/raw/NETDUINO3_WIFI/latest/) | STM32F427VI   | 168 MHz   | 1408kB  | 164kB  |
+| [BrainPad2](ChibiOS/BrainPad2) |  [![Latest Version @ Cloudsmith](https://api-prd.cloudsmith.io/v1/badges/version/net-nanoframework/nanoframework-images-community-targets/raw/BrainPad2/latest/x/?render=true)](https://cloudsmith.io/~net-nanoframework/repos/nanoframework-images-community-targets/packages/detail/raw/BrainPad2/latest/) | STM32F401RE   | 84 MHz   | 512 kB  | 96 kB  |
```
Comment (josesimoes) [inline README.md:33]: Please extract this to another PR.

File: README.md [hunk 2]
```diff
@@ -52,7 +53,8 @@ The above firmware builds include support for the class libraries and features m
...
   | PybStick Std 26          | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |  |                    | :heavy_check_mark: |                    |  |  |                    |
-  | NETDUINO3_WIFI          | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |                    | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |                    |                    |                    |                    |                    |
+  | NETDUINO3_WIFI           | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |                    | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |                    |                    |                    |                    |                    |
```
Comment (josesimoes) [inline README.md:56]: Please extract this to another PR.

File: README.md [hunk 3]
```diff
@@ -52,7 +53,8 @@ The above firmware builds include support for the class libraries and features m
...
+  | BrainPad2                | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |                    | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |                    |                    |                    |                    |                    |
```
Comment (josesimoes) [inline README.md:57]: Please extract this to another PR.

File: README.md (hunk 1 above)
Comment (networkfusion) [reply README.md:33]: Probably easier to do it the other way around 😁
